### DISCUSSION TITLE
How to change exporter `RetryPolicy`

### DISCUSSION THREAD

**Discussion by GalaxyiPhone** [open-telemetry/opentelemetry-java-instrumentation#12926] [Q&A]
I'm using opentelemetry spring boot starter.

I want to change the retry policy according to Span, Log transmission failure to prevent errors from accumulating in production.
Currently, the default value is 5 retries, but I want to modify it to 1 retry. Is it possible?

There doesn't seem to be any support through property, is it possible to register a bean?

**Comment by trask** [comment]
cc @jeanbisutti @zeitlinger

**Comment by zeitlinger** [comment]
You can use `otel.java.exporter.otlp.retry.disabled` to disable retries (same as in agent)
